src/Exportable.php

```diff
@@ -112,6 +112,7 @@ private function exportOrDownload($path, $function, callable $callback = null)
             }
         }
         $writer->close();
+        exit;
```

Comment:
We definitely can not exit script here. It breaks Laravel application flow such as middleware processing and terminate callbacks.